util/shlib_wrap.sh.in

```diff
@@ -98,8 +102,8 @@ NONSTOP_KERNEL)
 	SHLIB_PATH="${THERE}:$SHLIB_PATH"		# legacy HP-UX
 	LIBPATH="${THERE}:$LIBPATH"			# AIX, OS/2
 	export LD_LIBRARY_PATH DYLD_LIBRARY_PATH SHLIB_PATH LIBPATH
-	# Even though $PATH is adjusted [for Windows sake], it doesn't
-	# necessarily does the trick. Trouble is that with introduction
+	# Even though $PATH is adjusted [for Windows's sake], it doesn't
```

Comment:
Thanks @tom-cosgrove-arm I'll work with the OP to get this part added to the other PR, since it fits there

@GamingFennex based on the feedback, probably makes sense to close this PR